common/src/main/resources/assets/figura/lang/en_us.json

```
@@ -1541,13 +1542,13 @@
     "figura.docs.vector_n.div": "Divides this vector by the given vector or values, and returns self for chaining",
     "figura.docs.vector_n.reduce": "Reduces this vector modulo the given vector or values, and returns self for chaining",
     "figura.docs.vector_n.scale": "Scales this vector by the given factor, and returns self for chaining",
-    "figura.docs.vector_n.unpack": "Returns each of this vector values as argument",
+    "figura.docs.vector_n.unpack": "Returns each of this vector values as an argument",
```

This might be clearer.
```suggestion
    "figura.docs.vector_n.unpack": "Returns this vector's values as separate numbers",
```